Source/SaveExtension/Public/SaveFilter.h

```diff
@@ -55,15 +55,23 @@ struct FSaveFilter
 	FSaveFilter() {}
 	FSaveFilter(const USavePreset& Preset)
 	{
-		ActorFilter         = Preset.GetActorFilter(true);
-		ComponentFilter     = Preset.GetComponentFilter(true);
-		LoadActorFilter     = Preset.GetActorFilter(false);
+		ActorFilter         = Preset.GetActorFilter(true);	
+		ComponentFilter     = Preset.GetComponentFilter(true);		
+		LoadActorFilter     = Preset.GetActorFilter(false);		
 		LoadComponentFilter = Preset.GetComponentFilter(false);
-
+		BakeAllowedClasses();
 		MaxFrameMs       = Preset.GetMaxFrameMs();
 		bStoreComponents = Preset.bStoreComponents;
 	}
 
+	FORCEINLINE void BakeAllowedClasses()
```

Comment:
This function is long enough so that it's not worth inlining.
The easy rule to follow is to only inline one-liners.

Reply:
done